sql/core/src/test/resources/sql-tests/inputs/change-column.sql

```diff
@@ -0,0 +1,53 @@
+-- Create the origin table
+CREATE TABLE test_change(a Int, b String, c Int);
```

Comment:
nit: in SQL statement, we should upper case the type string, i.e. `STRING`, `INT`, etc.